scripts/push_ahu_setpoints.py

```
@@ -0,0 +1,120 @@
+    if len(extra) != len(wanted):
+        found = {r.get("point_id") for r in extra}
+        missing = sorted(wanted - found)
+        print(f"Expected {len(wanted)} rows in {csv_path}, matched {len(extra)}; missing: {missing}", file=sys.stderr)
+        return 1
+
+    base = f"http://{args.host}"
+    token = _login(base, args.user, password)
+    headers = {"Authorization": f"Bearer {token}", "Content-Type": "application/json"}
+
+    body = json.dumps({"rows": extra, "enable_poll": True}).encode()
+    req = urllib.request.Request(f"{base}/api/bacnet/driver/merge-rows", data=body, headers=headers, method="POST")
```

Ruff (0.15.15)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)